components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -642,6 +642,18 @@ def leave_none_unset_int(s):
          ("Whether or not to include a user dropdown in the base template."
           " Particularly useful when used in combination with the OMERO.web"
           " public user where logging in may not make sense.")],
+    "omero.web.feedback.comment.enable":
```

Comment:
Unfortunately, we do have a mixture of `<property>.enable` and `<property>.enabled` but the recently added properties seem to comply with the second scheme. Thought about using `.enabled` instead?